### ISSUE TITLE
predict() in Predict.py function issue "only one element tensors can be converted to Python scalars"

### ISSUE THREAD

**Issue by hlian188** [ProsusAI/finBERT#20]
When I call predict, I get the error "only one element tensors can be converted to Python scalars" on line 618 of finbert.py.
When I modify the line from:
`logits = softmax(np.array(logits))`
to
`logits = softmax(np.array(logits[0]))`

I get no error, but the predictions and sentiment scores do not seem right when I tested it on examples.csv. The logit looks like [small number, .99..., small number], so the labels are all negative and the scores are all around -.99...

For reference, I copied predict.py, finbert.py, and util.py into a jupyter notebook and used the following as my model
`model = BertForSequenceClassification.from_pretrained("ipuneetrathore/bert-base-cased-finetuned finBERT",num_labels=3,cache_dir=None)
`

**Comment by doguaraci** [comment]
I'm not sure what might be causing the problem. Could you please post the code that is causing the error and print `logits` right before you get the error? And also the complete error message.

**Comment by sharebear1993** [comment]
I also get the same error. Please see the code I used and the logits before the error

![image](https://user-images.githubusercontent.com/13189440/89350923-f00f9d80-d67e-11ea-8661-7c046d630309.png)
![image](https://user-images.githubusercontent.com/13189440/89351051-3533cf80-d67f-11ea-823b-d3a750d0be8f.png)
![image](https://user-images.githubusercontent.com/13189440/89351685-855f6180-d680-11ea-8106-5a0faafcde90.png)

**Comment by sharebear1993** [comment]
When I modify the code like hlian1888 (logits = softmax(np.array(logits[0]))), I get these results which are different from the notebook on github

My results
![image](https://user-images.githubusercontent.com/13189440/89352126-6e6d3f00-d681-11ea-9852-d8d256b48b6d.png)

**Comment by doguaraci** [comment]
@sharebear1993 sorry for the very late response. This problem should be fixed with the latest commit.
